punchr.proto

```diff
@@ -173,3 +179,24 @@ enum HolePunchAttemptOutcome {
   // The hole punch was performed and was successful
   HOLE_PUNCH_ATTEMPT_OUTCOME_SUCCESS = 6;
 }
+
+
+message RelayInfo {
+  // The relay peer ID
+  required bytes relay_id = 1;
+
+  // The relay agent_version
+  required string agent_version = 2;
+
+  // The relay protocols
+  repeated string protocols = 3;
+
+  // The relay multi address
+  repeated bytes relay_multi_addresses = 4;
+
+  // rtt measurement to the relay
+  required float rtt = 5;
+
+  // estimated rtt from the remote peer to the relay
+  required float estimated_remote_rtt = 6;
```

Comment:
Let `remote_rtt_through_relay` be the rtt from our node to the remote on the relayed connection. If I see it correctly then
`estimated_remote_rtt` = `remote_rtt_through_relay` - `rtt`?
If that's the case, how about we just add `remote_rtt_through_relay` directly as property here, and let the server do the calculation? At least on rust we don't have the `estimated_remote_rtt` properties, but just from the ping protocol the rtt to the relay and the rtt on the relayed connection to the remote.

Reply:
It would be great to also be on the rust client 😄 

True, the calculation could indeed be done in the server, and does make more sense.